src/ucp/proto/proto_common.h

```diff
@@ -253,13 +253,22 @@ ucp_proto_common_find_lanes(const ucp_proto_common_init_params_t *params,
                             ucp_lane_index_t *lanes);
 
 
+ucp_lane_index_t ucp_proto_common_find_lanes_internal(
```

Comment:
Maybe it worth renaming? Defining this func in header seems to make `_internal()` suffix redundant FMPOV. WDYTA renaming this to `ucp_proto_common_find_lanes` and renaming origin `ucp_proto_common_find_lanes` to `ucp_proto_common_find_lanes_check_frag`?